src/main/java/com/amadeus/Amadeus.java

```diff
@@ -25,7 +30,7 @@
   /**
    * The API version.
    */
-  public static final String VERSION = "5.6.0";
+  public static final String VERSION = Amadeus.getSystemProperties(Constants.VERSION_NAME);
```

Comment:
Based on the other comment, when an exception is thrown we would like to capture it, stop the process and not version it.

Reply:
In fact that's true. The above method could reach any defined property. I think you are right it's better throw the exception.

So you suggest just to cause system exit on exception?

Reply:
Hello @SealSoft, we had another look at the code and instead of system exit on exception it could be better to return "N/A". That's because it seems the AMADEUS.VERSION doesn't affect the release and it is used only as user agent information in the API [request](https://github.com/amadeus4dev/amadeus-java/blob/master/src/main/java/com/amadeus/Request.java#L99). So even if it's "N/A" the release and API calls won't fail.

Hope that's clear and let us know if you have any questions.